Description: skip part of "framing_offset_size_bump" test on 32-bit.
 skip part of a test on 32-bit. The test fails with an overflow error. While
 this is arguablly a bug, I belive in reality the program will run out of
 address space before it becomes a real issue.

 In any case, this is a legacy package, and is not a regression compared to
 what is currently in testing. Skipping this test will allow the rest of the
 testsuite to be useful on 32-bit architectures.

Author: Peter Michael Green <plugwash@debian.org>

--- rust-zvariant-2-2.10.0.orig/src/framing_offset_size.rs
+++ rust-zvariant-2-2.10.0/src/framing_offset_size.rs
@@ -111,6 +111,7 @@ mod tests {
             FramingOffsetSize::for_bare_container(std::u32::MAX as usize - 12, 3),
             FramingOffsetSize::U32
         );
+        #[cfg(target_pointer_bits = "64")]
         assert_eq!(
             FramingOffsetSize::for_bare_container(std::u32::MAX as usize - 11, 3),
             FramingOffsetSize::U64
